Microsoft.DotNet.Interactive.Js/tests/Responses/variableBundleResponse01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "csharp":
+  {
+      "a": 1,
+      "b": "2"
+  },
+  "fsharp": {
+      "c": 3,
+      "d": false
+  }
+}
```

Comment:
Can we give this file a more intention-revealing name?

Do these responses need to be in separate files?